lib/prometheus-parser/src/line.rs

```diff
@@ -140,6 +145,13 @@ impl Metric {
         })
     }
 
+    fn parse_timestamp(input: &str) -> IResult<Option<i64>> {
```

Comment:
```suggestion
    fn(crate) parse_timestamp(input: &str) -> IResult<Option<i64>> {
```

In line with the other changes.

Reply:
Whoops, wrong